recipe/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "trulens-core" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trulens_core-{{ version }}.tar.gz
+  sha256: bfbadd53a5a5d17575a8855f3fd888cad47912eb53fc4b0f27487ad341ba9971
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 100
+  skip: true  # [py<39]
+
+requirements:
+  host:
+    - python
```

```suggestion
    - python !=3.9.7
```
Should we add something like this explicitly because of [this](https://github.com/truera/trulens/blob/trulens-1.0.7/pyproject.toml#L30)?

---

Great catch, yes I do think we should add that.